locals.tf

```diff
@@ -7,8 +7,7 @@ locals {
   ingress_sg_sg_rule_description         = var.ingress_sg_sg_rule_description
   name                                   = var.name
   replication_group_description          = coalesce(var.replication_group_description, "Replication group for ${local.name}, managed by terraform")
-  route53_record_name                    = "${local.name}-CNAME"
-  route53_record_entry                   = "${local.name}.${replace(data.aws_route53_zone.selected.name, "/[.]$/", "")}"
+  route53_record_name                    = local.create_route_53_cname_record ? "${local.name}-CNAME" : ""
```

Comment:
I still don't understand what this is here for? You don't appear to use it anywhere.

Reply:
:+1: done deleted it